queue_services/account-mailer/src/account_mailer/worker.py

```diff
@@ -227,3 +279,72 @@ async def cb_subscription_handler(msg: nats.aio.client.Msg):
     except Exception:  # NOQA # pylint: disable=broad-except
         # Catch Exception so that any error is still caught and the message is removed from the queue
         logger.error('Queue Error: %s', json.dumps(event_message), exc_info=True)
+
+
+def map_templates_based_on_message_type(message_type: str):
+    """Provide template, subject and args based on message type."""
```

Comment:
this is very good ..I really like this code..
just wondering ; may be we just assume the template name from message type and avoid all the ifs? is there a way?